src/Common/Auth/OneTimeAuth.php

```diff
@@ -68,7 +68,7 @@ public function createPortalOneTime(array $p, bool $encrypt_redirect = false): a
         $date_base = ($p['enabled_datetime'] ?? null) ?: 'NOW';
         $expiry = new DateTime($date_base);
         $expiry->add(new DateInterval($p['expiry_interval'] ?? 'PT15M'));
-        $token_raw = RandomGenUtils::createUniqueToken(32);
+        $token_raw = RandomGenUtils::createUniqueToken(16);
```

Comment:
is there any check or mechanism in place to prevent storing a duplicate one time raw token?